asyncapi/slack/service_types.bal

```diff
@@ -14,20 +19,29 @@ public type SlackEventsChannelService service object {
     remote function onChannelUnarchive(GenericEventWrapper payload) returns error?;
 };
 
-public type SlackEventsDndService service object {
+# Triggers when a new event is received related to Slack do not disturb (DND) settings. 
```

Comment:
Suggestion `Slack Do Not Disturb (DND)..`

Reply:
Thanks a lot for the suggestion. Resolved